From e723dbaf4586ec773a9cfef8a379bf0ca1751b58 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Mon, 5 Aug 2019 12:01:14 +0900
Subject: [PATCH] spp-ctl: rename classifier_mac to classifier

As renamed classifier_mac in spp_vf, this update is to rename in
spp-ctl.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/spp-ctl/spp_webapi.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/spp-ctl/spp_webapi.py b/src/spp-ctl/spp_webapi.py
index 25b2e6f..8451711 100644
--- a/src/spp-ctl/spp_webapi.py
+++ b/src/spp-ctl/spp_webapi.py
@@ -243,7 +243,7 @@ class V1VFHandler(BaseHandler, V1VFCommon):
         return self.convert_info(proc.get_status())
 
     def vf_comp_start(self, proc, body):
-        self.validate_comp_start(body, ["forward", "merge", "classifier_mac"])
+        self.validate_comp_start(body, ["forward", "merge", "classifier"])
         proc.start_component(body['name'], body['core'], body['type'])
 
     def vf_comp_stop(self, proc, name):
-- 
2.17.1

